src/components/TransactionStatusMock.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
i wonder if we need another component for this, can we try to add the initial values on `TransactionStatus` component? maybe i'm missing a reason for not do so.

Reply:
The TransactionStatus components changes the transactions through the relevant hook/context. I tried to use that but there is no way that you can dispatch the transaction and replace the empty one instantly. In order to avoid to remove all the dispatch/reducer logic just in order to show an empty transaction - I preferred to make a simple Mock Component that will be destroyed as soon as transactions will start flow.

Reply:
everything clear.... go ahead with this.

Reply:
Go ahead is not enough :) Approval is missing :P